### DISCUSSION TITLE
Problem with ordering of media queries (max-width)

### DISCUSSION THREAD

**Discussion by Haller125** [tailwindlabs/tailwindcss#13499] [Help] [answered]
![image](https://github.com/tailwindlabs/tailwindcss/assets/65864303/aece7a8c-d4dc-4b12-bb42-e7a3f8eddd53)

As you can see from screenshot, rule for 4.5s is on the top, so rules for other breakpoiints doesn't work. Is there some way to handle with it? 

I tried to change order in both config and jsx element but it didn't work.

Config:
theme: {
    screens: {
      '1s' : {'max':'576px'},
      '2s' : {'max':'768px'},
      '3s' : {'max':'992px'},
      '3.5s': {'max':'1200px'},
      '4s' : {'max':'1400px'},
      '4.5s': {'max':'1600px'},
    },
}

Part of code:
`<h1 className="font-extralight text-white text-7xl 4.5s:text-[64px] 4s:text-[50px] 3.5s:text-[40px] 3s:text-[36px] 2s:text-[30px]">Some Text</h1>`

**Comment by alex-krasikau** [comment, marked as answer]
Hey @Haller125!

You need to reorder the definitions in the theme object like this:

```js
theme: {
  screens: {
    '4.5s': { max: '1600px' },
    '4s': { max: '1400px' },
    '3.5s': { max: '1200px' },
    '3s': { max: '992px' },
    '2s': { max: '768px' },
    '1s': { max: '576px' },
  },
}
```

Check out this complete example - https://play.tailwindcss.com/UdgOlTlZLI?file=config

**Reply by Haller125** [reply]
Wow! Thank you!
